codalab/server/bundle_manager.py

```diff
@@ -29,16 +29,11 @@ class BundleManager(object):
     Assigns run bundles to workers and makes make bundles.
     """
 
-    @staticmethod
-    def create(codalab_manager):
+    def __init__(self, codalab_manager):
         config = codalab_manager.config.get('workers')
         if not config:
             print('config.json file missing a workers section.', file=sys.stderr)
-            exit(1)
-
-        from codalab.server.default_bundle_manager import DefaultBundleManager
-
-        self = DefaultBundleManager()
+            sys.exit(1)
```

Comment:
[It's bad to use `exit()` to exit from programs.](https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python)